---
layout: page
title: N.W. Anderson
---
**The triumphant vindication of bold theories - are these not the pride and the justification of our life's work?**
~ *Sherlock Holmes*

<hr color = '#fff'> 

### Summary

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; I am currently developing a diffusion model of background selection. Recent work includes path integral approaches to solve Kimura’s diffusion equation with complex (time and/or frequency dependent) selective pressures and time series statistical methods to study the role of epistasis in rapid adaptation, particularly during adaptation to climate change. I have a special interest in the rate of turnover in the genetic architecture of diseases in human populations. Previous work includes phylogenetic analysis of complex trait evolution and coevolution between traits, mathematical modeling of stochastic biological systems, development of  multilocus models of allele diffusion with ongoing migration, complex selective pressures and abnormal life histories.



<hr color = '#fff'>

### Publications

**Time and/or frequency dependent solution to Kimura's diffusion**

N. W. Anderson, L. Kirk, J. Schraiber and A. P. Ragsdale. A Path Integral Approach for Allele Frequency Dynamics Under Polygenic Selection. Genetics. 2024; ayaie182. <https://doi.org/10.1093/genetics/ayae182>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Many phenotypic traits have a polygenic genetic basis, making it challenging to learn their genetic architectures and predict individual phenotypes. One promising avenue to resolve the genetic basis of complex traits is through evolve-and-resequence experiments, in which laboratory populations are exposed to some selective pressure and trait-contributing loci are identified by extreme frequency changes over the course of the experiment. However, small laboratory populations will experience substantial random genetic drift, and it is difficult to determine whether selection played a roll in a given allele frequency change. Predicting how much allele frequencies change under drift and selection had remained an open problem well into the 21st century, even those contributing to simple, monogenic traits. Recently, there have been efforts to apply the path integral, a method borrowed from physics, to solve this problem. So far, this approach has been limited to genic selection, and is therefore inadequate to capture the complexity of quantitative, highly polygenic traits that are commonly studied. Here we extend one of these path integral methods, the perturbation approximation, to selection scenarios that are of interest to quantitative genetics. In particular, we derive analytic expressions for the transition probability (i.e., the probability that an allele will change in frequency from x , to y in time t ) of an allele contributing to a trait subject to stabilizing selection, as well as that of an allele contributing to a trait rapidly adapting to a new phenotypic optimum. We use these expressions to characterize the use of allele frequency change to test for selection, as well as explore optimal design choices for evolve-and-resequence experiments to uncover the genetic architecture of polygenic traits under selection. 

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/NW-Anderson/transition_probs)

**ABC based inference of epistasis**

Stern, D.B., N.W. Anderson, J.A. Diaz and C.E. Lee. Genome-wide signatures of synergistic epistasis during parallel adaptation in a Baltic Sea copepod. Nature Communications. 2022; 13:4024. <https://doi.org/10.1038/s41467-022-31622-8>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>The role of epistasis in driving adaptation has remained an unresolved problem dating back to the Evolutionary Synthesis. In particular, whether epistatic interactions among genes could promote parallel evolution remains unexplored. To address this problem, we employ an Evolve and Resequence (E&R) experiment, using the copepod Eurytemora affinis, to elucidate the evolutionary genomic response to rapid salinity decline. Rapid declines in coastal salinity at high latitudes are a predicted consequence of global climate change. Based on time-resolved pooled whole-genome sequencing, we uncover a remarkably parallel, polygenic response across ten replicate selection lines, with 79.4% of selected alleles shared between lines by the tenth generation of natural selection. Using extensive computer simulations of our experiment conditions, we find that this polygenic parallelism is consistent with positive synergistic epistasis among alleles, far more so than other mechanisms tested. Our study provides experimental and theoretical support for a novel mechanism promoting repeatable polygenic adaptation, a phenomenon that may be common for selection on complex physiological traits.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/TheDBStern/Baltic_Lab_Wild)

**Partitioning genetic architecture by dominance and mode of inheritance** 

Armstrong A, N. W. Anderson, H. Blackmon. Inferring the potentially complex genetic architectures of adaptation, sexual dimorphism, and genotype by environment interactions by partitioning of mean phenotypes. Journal of Evolutionary Biology. 2019; 32:4 369-379. <https://doi.org/10.1111/jeb.13421>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Genetic architecture fundamentally affects the way that traits evolve. However, the mapping of genotype to phenotype includes complex interactions with the environment or even the sex of an organism that can modulate the expressed phenotype. Line cross analysis is a powerful quantitative genetics method to infer genetic architecture by analyzing the mean phenotype value of two diverged strains and a series of subsequent crosses and backcrosses. However, it has been difficult to account for complex interactions with the environment or sex within this framework. We have developed extensions to line cross analysis that allow for gene by environment and gene by sex interactions. Using extensive simulations studies and reanalysis of empirical data, we show that our approach can account for both unintended environmental variation when crosses cannot be reared in a common garden and can be used to test for the presence of gene by environment or gene by sex interactions. In analyses that fail to account for environmental variation between crosses we find that line cross analysis has low power and high false positive rates. However, we illustrate that accounting for environmental variation allows for the inference of adaptive divergence, and that accounting for sex differences in phenotypes allows practitioners to infer the genetic architecture of sexual dimorphism.</q> 

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/coleoguy/SAGA2)

**Probability of Sex-Autosome Fusions**

Anderson N.W., C. E. Hjelmen and H. Blackmon. The Probability of Fusions Joining Sex Chromosomes and Autosomes. Biology Letters. 2020; 16:11. <https://doi.org/10.1098/rsbl.2020.0648>


&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Chromosome fusion and fission are primary mechanisms of karyotype evolution. In particular, the fusion of a sex chromosome and an autosome has been proposed as a mechanism to resolve intralocus sexual antagonism. If sexual antagonism is common throughout the genome, we should expect to see an excess of fusions that join sex chromosomes and autosomes. Here, we present a null model that provides the probability of a sex chromosome autosome fusion, assuming all chromosomes have an equal probability of being involved in a fusion. This closed-form expression is applicable to both male and female heterogametic sex chromosome systems and can accommodate unequal proportions of fusions originating in males and females. We find that over 25% of all chromosomal fusions are expected to join a sex chromosome and an autosome whenever the diploid autosome count is fewer than 16, regardless of sex chromosome system. We also demonstrate the utility of our model by analyzing two contrasting empirical datasets: one from Drosophila and one from the jumping spider genus Habronattus. We find that in the case of Habronattus there is a significant excess of sex chromosome autosome fusions but that in Drosophila there are far fewer sex chromosome autosome fusions than would be expected under our null model.
 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/coleoguy/Sex-Autosome-Fusion)

<hr color = '#fff'>

### Current Projects

Demographic inference methods rely on allelic transition probabilities to compute the expected allele frequency spectrum of a given demographic model. However, many solutions assume neutrality or selection acting on the focal loci - implicitly assuming that each locus evolves independently. However, linked selection is a thing that exists, and weak, linked purifying selection (background selection) is ubiquitous and leads to significant bias in inferred demographic histories. I am addressing this problem using a diffusion approach to jointly model the evolution of a focal allele as well as the genomic backgrounds that it appears upon.

<hr color = '#fff'> 

### Past Projects

**Ancestral Condition Test**

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Analyses of the co-evolution of multiple traits has the potential to reveal the drivers and limits to biological evolution. A variety of methods are available to study the interaction between either two continuous traits or a discrete trait that impacts the evolution of a continuous trait. However, few methods are available to study the impact of a continuous trait on the evolution of a discrete trait. Here we present the ancestral condition test, a new comparative method that evaluates whether a binary trait tends to transition when a continuous trait has values more extreme than expected if both traits were evolving independently. This approach leverages ancestral state estimates of both the continuous and the binary trait to test whether extreme values of the continuous trait are associated with transitions in the binary trait, and to assess statistical significance.  We explore the robustness of our approach under a range of parameter values and patterns of trait evolution. We find that either a relatively strong contingency between the two traits or a large number of taxa is required to detect the underlying relationships reliably. Statistical power of the test is highest when the binary trait evolves unidirectionally, and we find that the false-positive rate remains acceptable for a bidirectionally evolving binary trait. In comparison to existing methods that might be employed, we show that the ancestral condition test has both higher power and a lower false-positive rate. The types of questions that this approach allows us to test are common in evolutionary biology and, unlike existing methods, the ancestral condition test incorporates the temporal order of transitions – moving a step closer to inferring causality rather than merely identifying correlation. An implementation of this test is distributed in the r package evobiR.
 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://drive.google.com/open?id=1xWjilhFZ34JdqAH0Sq6QrtmwTxlXyfsW). <br> 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/NW-Anderson/Ancestral-Condition-Test).

**How much water is in the fountain of youth?** 

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Among species that have separate sexes, sex chromosomes are nearly ubiquitous and yet there are many unanswered question with regard to their evolutionary dynamics. XY sex chromosome systems are one of the most common methods of sex determination and have long interested researchers. Normally X and Y chromosomes differentiate over time as the Y chromosome decays. However, not all species experience this Y decay. The fountain of youth hypothesis suggests that imperfect sexual development and deleterious mutations on Y chromosomes may act together as a force to maintain homology between the X and Y. However, the viability of the fountain of youth hypothesis has not been well explored mathematically. Using both finite and infinite population genetic models we have shown that this process cannot completely eliminate sexually antagonistic selection – the force that is thought to lead to the decay of Y chromosomes. Using our modeling approach we are able to determine the parameter space under which the fountain of youth can and cannot preserve similarity between the X and Y chromosome. These results appear to support fountain of youth hypothesis by showing the limits to the canonical model of sex chromosome evolution and grant insight into the fitness effect of sex chromosome inversions.</q>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://drive.google.com/file/d/1z3TgDcqhsfpzkswb43SNzQeYvo4kCNtR/view?usp=sharing). <br> 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github.](https://github.com/NW-Anderson/FOY)

**The Temporal Contingency Test: discovering correlation in the evolution of discrete traits**

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>One of the central questions of evolutionary biology is how traits interact with each other over the course of evolution.  However, most of the methods available for understanding correlation or contingency in the evolution of discrete traits are based on detecting differences in the rate of transitions in one trait when it is associated with a specific state of the other trait.  These methods lead to several known problems.  We solve these problems by developing an approach that focuses on temporal contingencies in the transitions of discrete traits.  Our statistical approach can determine whether transitions in one trait lead to transitions in a second trait more quickly than would be expected under a model where the two traits evolve independently.</q>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://docs.google.com/presentation/d/1pk9tAbmJ4eOlU8Y5A-5d2FPTz_BxWOH-tNLFeFKgdCU/edit?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/NW-Anderson/TempCorr).


<hr color = '#fff'>
