---
layout: page
title: Who is ready to make some science?
---
~ *Cave Johnson*

<hr color = '#fff'> 



&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; I have been a researcher in the lab of Dr. Heath Blackmon (Dept. of Biology) since 2017. The lab has a broad focus in evolutionary biology but much of the research focuses on sex chromosomes, genome structure, and population genetics. My work has focused on recombination rate evolution, phylogenetics and population genetic modelling, statistical methods for analysis of of trait evolution. Namely, the development of multi locus simulation of sexually antagonistic alleles under complex sex determination and recombination systems, and the statistical analysis of discrete and continuous trait evolution in a phylogenetic framework. I also lead a large scale experimental evolution project studying recombination rate adaptation to multilocus selection and changes in population structure. 

<hr color = '#fff'>

### Publications

**SAGA2.0** 

Armstrong A, N. W. Anderson, H. Blackmon. Inferring the potentially complex genetic architectures of adaptation, sexual dimorphism, and genotype by environment interactions by partitioning of mean phenotypes. Journal of Evolutionary Biology. 2019; 32:4 369-379. <https://doi.org/10.1111/jeb.13421>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Genetic architecture fundamentally affects the way that traits evolve. However, the mapping of genotype to phenotype includes complex interactions with the environment or even the sex of an organism that can modulate the expressed phenotype. Line cross analysis is a powerful quantitative genetics method to infer genetic architecture by analyzing the mean phenotype value of two diverged strains and a series of subsequent crosses and backcrosses. However, it has been difficult to account for complex interactions with the environment or sex within this framework. We have developed extensions to line cross analysis that allow for gene by environment and gene by sex interactions. Using extensive simulations studies and reanalysis of empirical data, we show that our approach can account for both unintended environmental variation when crosses cannot be reared in a common garden and can be used to test for the presence of gene by environment or gene by sex interactions. In analyses that fail to account for environmental variation between crosses we find that line cross analysis has low power and high false positive rates. However, we illustrate that accounting for environmental variation allows for the inference of adaptive divergence, and that accounting for sex differences in phenotypes allows practitioners to infer the genetic architecture of sexual dimorphism.</q> 

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/coleoguy/SAGA2)

**Ancestral Condition Test**

Anderson N. W, R. H. Adams, J. P. Demuth, H. Blackmon. Assessing the Impact of Continuous Traits on the Evolution of Discrete Traits: The Ancestral Condition Test. Methods in Ecology and Evolution. In Review.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Analyses of the co-evolution of multiple traits has the potential to reveal the drivers and limits to biological evolution. A variety of methods are available to study the interaction between either two continuous traits or a discrete trait that impacts the evolution of a continuous trait. However, few methods are available to study the impact of a continuous trait on the evolution of a discrete trait. Here we present the ancestral condition test, a new comparative method that evaluates whether a binary trait tends to transition when a continuous trait has values more extreme than expected if both traits were evolving independently. This approach leverages ancestral state estimates of both the continuous and the binary trait to test whether extreme values of the continuous trait are associated with transitions in the binary trait, and to assess statistical significance.  We explore the robustness of our approach under a range of parameter values and patterns of trait evolution. We find that either a relatively strong contingency between the two traits or a large number of taxa is required to detect the underlying relationships reliably. Statistical power of the test is highest when the binary trait evolves unidirectionally, and we find that the false-positive rate remains acceptable for a bidirectionally evolving binary trait. In comparison to existing methods that might be employed, we show that the ancestral condition test has both higher power and a lower false-positive rate. The types of questions that this approach allows us to test are common in evolutionary biology and, unlike existing methods, the ancestral condition test incorporates the temporal order of transitions – moving a step closer to inferring causality rather than merely identifying correlation. An implementation of this test is distributed in the r package evobiR.
 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://drive.google.com/open?id=1xWjilhFZ34JdqAH0Sq6QrtmwTxlXyfsW). <br> 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/NW-Anderson/Ancestral-Condition-Test).&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

<hr color = '#fff'>

### Current Projects

**How much water is in the fountain of youth?** 

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Among species that have separate sexes, sex chromosomes are nearly ubiquitous and yet there are many unanswered question with regard to their evolutionary dynamics. XY sex chromosome systems are one of the most common methods of sex determination and have long interested researchers. Normally X and Y chromosomes differentiate over time as the Y chromosome decays. However, not all species experience this Y decay. The fountain of youth hypothesis suggests that imperfect sexual development and deleterious mutations on Y chromosomes may act together as a force to maintain homology between the X and Y. However, the viability of the fountain of youth hypothesis has not been well explored mathematically. Using both finite and infinite population genetic models we have shown that this process cannot completely eliminate sexually antagonistic selection – the force that is thought to lead to the decay of Y chromosomes. Using our modeling approach we are able to determine the parameter space under which the fountain of youth can and cannot preserve similarity between the X and Y chromosome. These results appear to support fountain of youth hypothesis by showing the limits to the canonical model of sex chromosome evolution and grant insight into the fitness effect of sex chromosome inversions.</q>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://drive.google.com/file/d/1z3TgDcqhsfpzkswb43SNzQeYvo4kCNtR/view?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github.](https://github.com/NW-Anderson/FOY)

**The Temporal Contingency Test: discovering correlation in the evolution of discrete traits**

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>One of the central questions of evolutionary biology is how traits interact with each other over the course of evolution.  However, most of the methods available for understanding correlation or contingency in the evolution of discrete traits are based on detecting differences in the rate of transitions in one trait when it is associated with a specific state of the other trait.  These methods lead to several known problems.  We solve these problems by developing an approach that focuses on temporal contingencies in the transitions of discrete traits.  Our statistical approach can determine whether transitions in one trait lead to transitions in a second trait more quickly than would be expected under a model where the two traits evolve independently.</q>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Poster](https://docs.google.com/presentation/d/1pk9tAbmJ4eOlU8Y5A-5d2FPTz_BxWOH-tNLFeFKgdCU/edit?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Talk](https://docs.google.com/presentation/d/1Pwcj7orPf9Db11KuQuzQ_T8qJb2X_6rngYaSdsLDUtE/edit?usp=sharing). <br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Github](https://github.com/NW-Anderson/TempCorr).

**Recombination Rate Response to Reductions in Population Size and Environmental Perturbations**

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<q style='display:inline-block; width:1000px; margin-left: 32px'>Recombination is an important force that impacts response to selection1, genetic variation2, and the fate of introgressed genes3. Experimental evolution experiments have shown that recombination rate can respond to both direct selection4 and strong selection on unrelated traits5. Furthermore, theoretical studies demonstrate that small population size6 and multi-locus selection7 can lead to increased recombination rate. Indeed, indirect selection and small founding populations have been hypothesized as the source of differences in recombination rate between closely related island species of Drosophila8. Domesticated plants have shown a consistent increase in the recombination rate compared to their wild proginators9. Domestication involves both small population sizes and strong multi-locus selection. However, we lack a clear understanding of the respective contribution of small population size and selection acting individually, and any interaction when they cooccur, to observed increases in recombination rate. Further, long-term trends in recombination rates following change in environment and population size have yet to be empirically explored. Without understanding how recombination rates respond to selective forces, there is little hope of understanding how and why recombination rates vary across the tree of life.  The proposed experiment will allow me to quantify the impacts of population size and indirect selection on the evolution of recombination rates.</q>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Research Proposal](https://drive.google.com/file/d/1pQb7_S9QV1BuahAXSTACTqINgDutQd3r/view?usp=sharing). 

<hr color = '#fff'>
